documentation/modules/exploit/unix/webapp/drupal_drupalgeddon2.md

```diff
@@ -0,0 +1,101 @@
+## Intro
+
+> This module exploits a Drupal property injection in the Forms API.
```

Comment:
remove `>`

Reply:
That was my smartass quoting the the module description. :P

Reply:
That was my smartass quoting the module description. :P